gap_quantization/quantization.py

```diff
@@ -295,7 +296,7 @@ def collect_stats(self):
 
     def dump_activations(self, image_path, transforms, rounded=False, save_dir=None):
         if save_dir is None:
-            save_dir = os.path.join(self.cfg['save_folder'], 'activation_dump')
+            save_dir = os.path.join(self.cfg['save_folder'], 'activations_dump')
```

Comment:
Why have you renamed this directory? Is it important?

Reply:
Now it has the same name with directory in ReID-demo, so it will be easier to copy this folder. But if renaming is undesirable, I can change it back and note to copy just a content of a folder